java-manta-it/src/test/java/com/joyent/manta/client/MantaClientSigningIT.java

```diff
@@ -53,7 +56,7 @@ public void beforeClass() throws IOException {
 
         mantaClient = new MantaClient(config);
         testPathPrefix = IntegrationTestConfigContext.generateBasePath(config, this.getClass().getSimpleName());
-        mantaClient.putDirectory(testPathPrefix, null);
+        mantaClient.putDirectory(testPathPrefix, true);
```

Comment:
This test case can't be run on its own without changing this to recursive `putDirectory`